src/components/Intro.js

```diff
@@ -251,9 +251,9 @@ function Tools() {
 export const Intro = () => {
   return (
     <section className="mt-1 mb-14">
-      <h2 className="text-4xl md:text-5xl font-semibold tracking-wide mb-4">
+      <h1 className="text-4xl md:text-5xl font-semibold tracking-wide mb-4">
```

Comment:
A page must have an `h1`, and after that headings must be in order (only ever increase by at most 1).

https://dequeuniversity.com/rules/axe/4.1/page-has-heading-one https://dequeuniversity.com/rules/axe/4.0/heading-order